gui/resources.xml

```diff
@@ -538,34 +554,39 @@ Vocabulary
           <property name="icon-name">mypaint-duplicate-symbolic</property>
           <signal name="activate" handler="duplicate_layer_cb"/>
         </object>
+        <accelerator key="d" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="RenameLayer">
           <property name="label" translatable="yes" context="Menu→Layer (labels), Accel Editor (labels)">Rename Layer…</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Enter a new name for the current layer.</property>
           <signal name="activate" handler="rename_layer_cb"/>
         </object>
+        <accelerator key="r" modifiers="GDK_CONTROL_MASK"/>
```

Comment:
Krita uses "F2" as well. What would you suggest to move the scraps navigation to then?